plugins/logs/plugin.go

```diff
@@ -233,17 +237,22 @@ func Lookup(manager *plugins.Manager) *Plugin {
 
 // Start starts the plugin.
 func (p *Plugin) Start(ctx context.Context) error {
-	p.logInfo("Starting decision log uploader.")
-	go p.loop()
+	p.logInfo("Starting decision logger.")
+	// only start the loop if we have a service or plugin
+	if !p.localOnly() {
```

Comment:
Hmm.. yea, I kept it doing the same thing it was before. The only thing that it would do it handle reconfiguration which might need some attention with the new changes. Need to make sure discovery bundle configurations will still take affect.. this might have broken that.

Reply:
Makes sense. Perhaps just always run the loop then?